node/clients/checkout.ts

```diff
@@ -248,6 +265,18 @@ export class Checkout extends JanusClient {
   public getProfile = (email: string) =>
     this.get<CheckoutProfile>(this.routes.profile(email))
 
+  public setManualPrice = (
+    orderFormId: string,
+    itemIndex: number,
+    price: number
+  ) =>
+    this.put<CheckoutOrderForm>(
+      this.routes.setManualPrice(orderFormId, itemIndex),
+      {
+        price,
+      }
+    )
```

Comment:
this could be removed from this client, but we can do it later on another PR

Reply:
yep! I forgot it. Thanks!